javascript/javascript3/week3/shoppingCart/main.js

```diff
@@ -0,0 +1,114 @@
+class Product {
+    constructor(name, price) {
+        this.name = name;
+        this.price = price;
+    }
+    convertToCurrency(currency) {
+        if (currency.toLowerCase() === "dollar") {
+            const dollar = this.price * 0.14;
+            return dollar;
+        } else if (currency.toLowerCase() === "euro") {
+            const euro = this.price * 0.13;
+            return euro;
+        } else if (currency.toLowerCase() === "rupee") {
+            const rupee = this.price * 23.03;
+            return rupee;
+        }
+        return this.price;
+    }
+}
+const outputDiv = document.getElementById("output");
+class ShoppingCart {
+    constructor(products) {
+        this.products = products;
+    }
+    addProduct(product) {
+        this.products.push(product);
+    }
+    removeProduct(product) {
+        const matchedIndex = this.products.indexOf(product);
+        if (matchedIndex > -1) {
+            this.products.splice(matchedIndex, 1);
+        }
+    }
+    searchProduct(productName) {
+        const searchedItems = this.products.filter(requiredPro =>
+            requiredPro.name.toLowerCase().includes(productName.toLowerCase())
+        );
+        return searchedItems;
+    }
```

Comment:
in application when I delete all letters from Search Items input window it shows me all the products